glide.yaml

```diff
@@ -40,6 +40,10 @@ import:
   version: ^1.1.1
   subpackages:
   - gomock
+- package: github.com/hashicorp/packer
+  version: ^1.3.2
+  subpackages:
+  - builder/azure/pkcs12
```

Comment:
Will commit this later. This is needed for `pkcs12.Encode` as native lib does not implement this.

Reply:
we can generate `pem` instead of `pkcs12` to avoid conversion if that works